admin-dev/themes/new-theme/js/components/grid/extension/position-extension.ts

```diff
@@ -222,15 +222,13 @@ export default class PositionExtension {
     rowsData: Array<RowDatas>,
   ): Array<DNDPositions> {
     const regex = /^row_(\d+)_(\d+)$/;
-    const mapping = Array(rowsData.length).map(Object);
+    const mapping = Array(rowsData.length).fill(undefined).map(Object);
 
     for (let i = 0; i < rowsData.length; i += 1) {
-      const regexResult = <RegExpPositions>regex.exec(rowsData[i].rowMarker);
-
-      if (regexResult?.rowId && regexResult?.oldPosition) {
-        mapping[i].rowId = regexResult.rowId;
-        mapping[i].oldPosition = parseInt(regexResult.oldPosition, 10);
-      }
+      // @ts-ignore
```

Comment:
Ever try to fix this or, if you can cast something instead of ignoring

Casting is usually way better (even if it's bad) than breaking ts typecheck

Reply:
I changed this and made a little refacto it should be cleaner and more stable now